userguide/supervised-learning/random_forest_classifier.md

```diff
@@ -8,7 +8,7 @@ forests.
 
 ##### Introductory Example
 
-In this example, we will use the [Mushrooms dataset](https://archive.ics.uci.edu/ml/datasets/mushroom).
+In this example, we will use the [Mushrooms dataset](https://archive.ics.uci.edu/ml/datasets/mushroom).decision_tree_regression.md
```

Comment:
This looks like a typo.

Reply:
Thanks for the attention to detail - fixed!

Reply:
I still see the typo on this file.